src/test/java/com/puppycrawl/tools/checkstyle/DefaultLoggerTest.java

```diff
@@ -28,19 +28,24 @@
 import org.junit.Test;
 
 import com.puppycrawl.tools.checkstyle.api.AuditEvent;
+import com.puppycrawl.tools.checkstyle.api.LocalizedMessage;
 
 public class DefaultLoggerTest {
 
     @Test
     public void testCtor() throws UnsupportedEncodingException {
+        final LocalizedMessage addExceptionMessage = new LocalizedMessage(0,
+                Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.addException",
+                new String[] {"myfile"}, null,
+                LocalizedMessage.class, null);
```

Comment:
@subkrish , please move declaration of addExceptionMessage close to its usage.
Please apply the same fix for all other cases in this PR.